Header header                         # Timestamp and frame ID
perception_stack/FinalObject[] objects # List of fused objects in this frame
